Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -260,7 +260,8 @@ open class AVFoundationPlayback: Playback {
         if let player = player as? AVQueuePlayer {
             playerLooper = AVPlayerLooper(player: player, templateItem: item)
         }
-        player?.allowsExternalPlayback = true
+        
+        player?.allowsExternalPlayback = !options.bool(kDisableExternalPlayback, orElse: false)
```

Comment:
In my opinion, reading was hindered by the number of possible cases in this variable. Could we extract into a more expressive one, like an external var?
What do you think?

Reply:
Sure can! Done in bbe83474317c6ec466c521d7406e84cc38352535

Reply:
=]